plugins/plugin-docker/che-plugin-openshift-client/src/main/java/org/eclipse/che/plugin/openshift/client/OpenShiftConnector.java

```diff
@@ -1318,9 +1319,15 @@ private ContainerInfo createContainerInfo(Service svc,
         info.setNetworkSettings(networkSettings);
         info.setHostConfig(hostConfig);
         info.setImage(imageInfo.getConfig().getImage());
+        info.setState(getContainerState(pod));
         return info;
     }
 
+    private ContainerState getContainerState(final Pod pod) {
+        ContainerState state = new ContainerState();
+        state.setStatus(pod.getStatus().getPhase());
```

Comment:
+1 to use ContainerState. And since in a pod there may be more than one container I would prefer `getContainerState` to require the index of the container in the pod or change the method to `getPodContainersState` where the returned value is a collections of `ContainerState`.